app/experimenter/templates/base.html

```diff
@@ -41,7 +41,7 @@ <h3 class="pt-2">
           <div class="col text-right">
             <span class="fas fa-user"></span>
             {{ request.user }}
-            <a class="nocolorstyle" href="{% url "home" %}?owner={{ request.user.id }}&archived=on">
+            <a class="nocolorstyle" href="{% url "home" %}?owner={{ request.user.id }}&amp;archived=on">
```

Comment:
Does the "&" need to be encoded here since it's in an `href` and not a display element?

Reply:
All `&` need to be encoded.